TaskModules/powershell/TaskModuleSqlUtility/SqlPackageOnTargetMachines.ps1

```diff
@@ -468,7 +468,42 @@ function Execute-DacpacDeployment
     Write-Verbose -Verbose $sqlPackageArguments
     
     Write-Verbose "Executing command: $sqlPackage $sqlPackageArguments"
-    Execute-Command -FileName "$sqlPackage"  -Arguments $sqlPackageArguments
+    if($PSVersionTable.PSVersion.Major -lt 4)
+    {
+        Run-Command "`"$sqlPackage`" $sqlPackageArguments"
+    }
+    else
+    {
+        Execute-Command -FileName "$sqlPackage"  -Arguments $sqlPackageArguments
```

Comment:
Can you move this within Run-Command function. The choice to call using cmd.exe or Execute-command can be within Run-Command function.